{{cookiecutter.project_slug}}/installer.py

```diff
@@ -0,0 +1,70 @@
+#!/bin/python
+
+import subprocess as sp
+from argparse import ArgumentParser, BooleanOptionalAction
+
+
+def condacmd(cmd : str):
+    '''Wrapper for conda shell commands.
+    
+    '''
+    proc =  sp.run(f"conda {cmd}", shell=True, stdout=sp.PIPE, stderr=sp.STDOUT)
+    outstream = ''
+    if not proc.stdout is None:
+        outstream += proc.stdout.decode('utf-8')
+    if not proc.stderr is None:
+        outstream += proc.stderr.decode('utf-8')
```

Comment:
I think the code will never fall here, since sp is not capturing stderr, but rather sending it to sp.STDOUT